gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ObservationCommentMapper.java

```diff
@@ -96,21 +94,6 @@ private void setObservationComment(Observation observation, String text) {
         }
     }
 
-    private Meta createMeta() {
```

Comment:
Is this no longer needed?

Reply:
No they are not, meta and identifier fields are generated in the ResourceUtil class as it is repeated functionality across many classes

Reply:
I see, thanks!